README.md

```diff
@@ -1,33 +1,111 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-icons.svg)](https://www.npmjs.com/package/@polymer/iron-icons)
 [![Build status](https://travis-ci.org/PolymerElements/iron-icons.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-icons)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-icons)
 
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-icons)_
+## iron-icons
 
+`iron-icons` is a utility import that includes the definition for the
+`iron-icon` element, `iron-iconset-svg` element, as well as an import for the
+default icon set.
 
-## &lt;iron-icons&gt;
-
-`iron-icons` is a utility import that includes the definition for the `iron-icon` element, `iron-iconset-svg` element, as well as an import for the default icon set.
-
-The `iron-icons` directory also includes imports for additional icon sets that can be loaded into your project.
+The `iron-icons` directory also includes imports for additional icon sets that
+can be loaded into your project.
 
 Example loading icon set:
 
 ```html
-<link rel="import" href="../iron-icons/maps-icons.html">
+<script type="module">
+  import '@polymer/iron-icons/maps-icons.js';
+</script>
 ```
 
-To use an icon from one of these sets, first prefix your `iron-icon` with the icon set name, followed by a colon, ":", and then the icon id.
+To use an icon from one of these sets, first prefix your `iron-icon` with the
+icon set name, followed by a colon, ":", and then the icon id.
 
 Example using the directions-bus icon from the maps icon set:
 
 ```html
```

Comment:
This bit of the readme isn't needed, right? Either it's in the docs that we link to, or its in the proper demo section

Reply:
I've removed most of the docs here and just left the links.